packages/main/test/samples/Menu.sample.html

```diff
@@ -267,10 +269,80 @@ <h3>Menu with initial menu items and additional menu items fetching from a datab
 			let closeNode = document.createElement("ui5-menu-item");
 			closeNode.setAttribute("text", "Close");
 			menuListBusy.appendChild(closeNode);
-		}, 3000);
+		}, 2000);
 	}, { once: true });
 </script>
 	</xmp></pre>
 </section>
 
+<section>
+	<h3>Sub-menu with items fetching from a database</h3>  
+	<div class="snippet">
+		<ui5-button id="btnSubMenuListBusy" class="samples-margin">Open Menu</ui5-button> <br/>
+		<ui5-menu id="subMenuListBusy">
+			<ui5-menu-item text="New File" accessible-name="Opens a file explorer" additional-text="Ctrl+Alt+Shift+N" icon="add-document"></ui5-menu-item>
+			<ui5-menu-item text="Open" icon="open-folder"></ui5-menu-item>
+			<ui5-menu-item text="Save" icon="save" busy-delay="100" busy></ui5-menu-item>
+		</ui5-menu>
+		<script>
+			btnSubMenuListBusy.addEventListener("click", function(event) {
+				subMenuListBusy.showAt(btnSubMenuListBusy);
+			});
+
+			let feched = false;
+
+			subMenuListBusy.addEventListener("ui5-before-open", function(event) {
+				const item = event.detail.item;
+				if (item && item.text === "Save" && !feched) {
+					setTimeout(function() {
+						item.removeAttribute("busy");
+						item.removeAttribute("busy-delay");
+						let saveLocally = document.createElement("ui5-menu-item");
+						saveLocally.setAttribute("text", "Save Locally");
+						saveLocally.setAttribute("iocn", "save");
+						let saveCloud = document.createElement("ui5-menu-item");
+						saveCloud.setAttribute("text", "Save on Cloud");
+						saveCloud.setAttribute("iocn", "upload-to-cloud");
+						item.append(saveLocally, saveCloud);
+						feched = true;
+					}, 2000);
+				}
+			});
+		</script>
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+<ui5-button id="btnSubMenuListBusy" class="samples-margin">Open Menu</ui5-button> <br/>
+<ui5-menu id="subMenuListBusy">
+	<ui5-menu-item text="New File" accessible-name="Opens a file explorer" additional-text="Ctrl+Alt+Shift+N" icon="add-document"></ui5-menu-item>
+	<ui5-menu-item text="Open" icon="open-folder"></ui5-menu-item>
+	<ui5-menu-item text="Save" icon="save" busy-delay="100" busy></ui5-menu-item>
+</ui5-menu>
+<script>
+	btnSubMenuListBusy.addEventListener("click", function(event) {
+		subMenuListBusy.showAt(btnSubMenuListBusy);
+	});
+
+	let feched = false;
+
+	subMenuListBusy.addEventListener("ui5-before-open", function(event) {
+		const item = event.detail.item;
+		if (item && item.text === "Save" && !feched) {
+			setTimeout(function() {
+				item.removeAttribute("busy");
+				item.removeAttribute("busy-delay");
+				let saveLocally = document.createElement("ui5-menu-item");
+				saveLocally.setAttribute("text", "Save Locally");
+				saveLocally.setAttribute("iocn", "save");
```

Comment:
Did you test this example? Because "iocn" is not available as attribute and I doubt the "save"  and "upload-to-cloud" icons are displayed.

Reply:
The icons aren't displayed as you pointed out.